### DISCUSSION TITLE
Simplify working with dangerous-force-http

### DISCUSSION THREAD

**Discussion by dselyuzhitskiy** [ory/hydra#2627] [Q&A] [answered]
Hi!

We use Hydra for production and staging environment, and for some reasons in our staging environment we have a huge list of http callbacks. It is such uncomfortable to use --dangerous-allow-insecure-redirect-urls, especially when you get process list with long command string or when you need to add new http callback. I know that it is better for security reasons, but maybe it will be better to add another flag that allow all http callbacks ?

**Comment by vinckr** [comment, marked as answer]
I think the naming of the flag should give some indication why we are reluctant to make using it much easier:
--**dangerous**-allow-**insecure**-redirect-urls

Hydra is a very security-focused product, so we generally try to make doing insecure things inconvenient. 
Not because we hate the user, but because experience has shown that people tend to use the convenient method. 

That being said, if `--dangerous-allow-insecure-redirect-urls` is something you absolutely need to use in your staging environment and you already know of a reasonable path to amending the flag for your purpose, feel free to open up an issue with some more concrete implementation considerations.

**Reply by zepatrik** [reply]
:100:% agree, this is cumbersome to use for a reason.
